app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MediaRanker</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <header>
+      <h1>
+      <%= link_to "Media Ranker", homepages_path %>
+      <small>Ranking the Best of Everything</small>
+      </h1>
+      <% if flash.count > 0 %>
```

Comment:
It probably would be way better to insert `flash` outside of the `header` tag to match the model website